src/pages/projects/forms/ProjectDetailsForm.tsx

```diff
@@ -280,6 +312,10 @@ const ProjectDetailsForm: FC<Props> = ({ formik, project, isEdit }) => {
                       "features_networks",
                       !formik.values.features_networks,
                     );
+
+                    if (!formik.values.features_networks && !isEdit) {
+                      formik.setFieldValue("default_project_network", "none");
+                    }
```

Comment:
```suggestion
                    const featuresNetworks = !formik.values.features_networks;
                    formik.setFieldValue("features_networks", featuresNetworks);
                    if (featuresNetworks && !isEdit) {
                      formik.setFieldValue("default_project_network", "none");
                    }
```
I think we are introducing a race here with updating the value and immediately reading it in the line after. Better capture the new value and use it twice to avoid the race. This seems also easier to read.